canonical-web-and-design/vanilla-framework
# Number classes

## Done

Adds new heading classes with numbers instead of words (`p-heading-1`, `p-heading-2`, etc).
Existing class names still work but are marked deprecated until 3.0 (`p-heading--one`, etc).

Updates all examples to use new class names.

Fixes #1406 

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/
- Make sure new headings classes work as expected.
- Check heading examples (use direct example pages, CodePen uses live Vanilla version which doesn't have these changes yet)
